### DISCUSSION TITLE
Mesh refinement for fluid domain

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#122] [General discussion]
Hi,

Is HemoCell compatible with PALABOS's LBM mesh refinement capability? I need to do a large simulation but only a fraction of the domain contains cells. I was wondering if I can refine the mesh only in that region.

Mesh refinement capability is already implemented in PALABOS:
https://youtu.be/sFMXcg4SKBg

Thanks

**Comment by gzavo** [comment]
Hi,
This would be a very interesting project, I don't think anybody has tested this yet.
I see no technical problem with this, as long as the cells are not trespassing to a different grid resolution (including the boundary conditions). Small changes in the code might be needed.
